tableau_utilities/tableau_file/tableau_file.py

```diff
@@ -120,6 +120,7 @@ def sections(self):
         yield self.columns
         yield self.folders_common
         yield self.extract
+        yield self.metadata_records
```

Comment:
```suggestion
```
This section is sub-section of `connection`,
i.e. `self.connection.metadata_records`

Reply:
Done